components/log-viewer-webui/client/src/theme/utils.ts

```diff
@@ -0,0 +1,37 @@
+import {theme} from "antd";
+
+import {THEME_MODE} from "./typings";
+
+
+/**
+ * Detects Antd theme algorithm from system theme.
+ *
+ * @return The system theme algorithm.
```

Comment:
Not sure if the linting settings allow this, but ideally:

```suggestion
 * @return The Antd theme algorithm corresponding to the system theme.
```